backend/config/validate_config.py

```
@@ -0,0 +1,21 @@
+from core.common.exceptions import ValidationError
+
+VALID_ENVS = {"development", "staging", "production", "testing"}
+REQUIRED_SETTINGS = {"SECRET_KEY", "DATABASE_URL", "APP_ENV"}
+
+
+def validate_env_config(config: dict) -> None:
+	"""Validate required configuration keys and supported environment values."""
+	missing = [key for key in REQUIRED_SETTINGS if not config.get(key)]
+	if missing:
+		missing_list = ", ".join(sorted(missing))
+		raise ValidationError(f"Missing required configuration values: {missing_list}")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)